### ISSUE TITLE
Invalid generator for VS2019 x64

### ISSUE THREAD

**Issue by Helco** [lukka/CppBuildTasks#7]
In CMakeSettings.json, Visual Studio still uses the old style of specifying the generator architecture "Visual Studio 16 2019 Win64", while CMake does not support this style anymore and instead requests using "Visual Studio 16 2019" and additionally the parameter "-A Win64". 

The task runcmake seems to use the exact generator as written in the CMakeSettings.json, but it should detect VS2019 and higher and extract the architecture from the generator field. Otherwise the build on azure breaks with the error message:

```
Processing configuration: 'x64-Release'. 
  | :\Program Files\CMake\bin\cmake.exe" -G "Visual Studio 16 2019 Win64" -DCMAKE_BUILD_TYPE=RelWithDebInfo  d:\a\1\s |  
  | CMake Error: Could not create named generator Visual Studio 16 2019 Win64 |  
  |   |  
  | Generators |  
  | * Visual Studio 16 2019         = Generates Visual Studio 2019 project files. |  
  |                                   Use -A option to specify architecture.
```

**Comment by lukka** [comment]
@Helco thanks for the reported bugs, I released the fixed tasks, let me know!

**Comment by Helco** [comment]
@lukka both this fix and the one for #8 seem to work just fine, now it's only me blocking the build :)
Many thanks for the quick fixes!
